sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -5,6 +5,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Why are you using `Newtonsoft.Json` instead of `System.Text.Json`?

Reply:
`System.Text.Json` doesn't support Json Path.

Reply:
yes. So the alternative will be to re write the whole json tree :(
as this is a test framework that we don;t expect our users to use, then it should be ok